cluster-scope/base/core/namespaces/xdmod/namespace.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: xdmod
```

Comment:
Change the namespace to xdmod-svcacct so that it doesn't conflict with xdmod

Reply:
I think I will submit another PR and close this one.